release_util_script_decrypt_var.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Where is this script used?

Reply:
Added to file
# This script can be used to double check that an encrypted variable found in a file (e.g. gpg key id) is correct (e.g. no misspellings have occurred).
# It is the peer of 'release_util_script_encrypt_var.sh' - which is used for encrypting a variable (e.g. a gpg key id).